src/scripts/main.js

```diff
@@ -1,3 +1,15 @@
 'use strict';
 
-// write your code here
+const rows = document.querySelectorAll('tr');
+
+for (let i = 0; i < rows.length; i++) {
+  const row = document.querySelectorAll('tr')[i];
+
+  cloneColumn(row);
+}
+
+function cloneColumn(column) {
```

Comment:
The function name 'cloneColumn' is misleading as it takes a row as an argument. Consider renaming it to 'cloneSecondColumnInRow' or a similar name that better describes its purpose.